### DISCUSSION TITLE
More Documentation

### DISCUSSION THREAD

**Discussion by captaingerhard** [johnwalley/d3-simple-slider#195] [Ideas]
Hello,

first of all, I really like the slider. Especially the New York Times example was exactly what I searched for.

But more documentation on how to style the slider would be great (the handle, the track  line etc; also how to hide the ouline of the handle).
I didn't worked that much with d3.js yet, so maybe that's all obvious, but I think it would be helpful.

Best Regards,
Gerhard

**Comment by johnwalley** [comment]
Hi @captaingerhard, sorry for the late response. I agree some more documentation would be desirable. I'm hoping to carve out some dedicated  time to work on my open source projects soon.

It's really helpful to have some specific questions to answer when writing documentation so thanks for your input!
